commerce/domain/tax/service/classes/TaxServiceExtensionSampleForCanada.cls-meta.xml

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<ApexClass xmlns="http://soap.sforce.com/2006/04/metadata">
+    <apiVersion>58.0</apiVersion>
```

Comment:
This must be 59.0 for 246 release.

Reply:
Pricing was using 58.0, so i got confused. will change it to 59.0